sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -139,6 +139,16 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    parent_parser.add_argument(
+        '--auth_user',
+        type=str,
+        help='username to authorize request')
```

Comment:
This help text should probably be clear this is to support Basic Auth if the REST API is using it.